manifests/pipeline.yaml

```diff
@@ -1,53 +1,67 @@
 apiVersion: v1
 kind: Template
 labels:
-  app: fedora-coreos
-  template: fedora-coreos-template
+  app: ${DISTRO}-coreos
+  template: ${DISTRO}-coreos-template
 metadata:
   annotations:
     description: |-
-      Jenkins pipeline for Fedora CoreOS.
+      Jenkins pipeline for CoreOS distributions.
     iconClass: icon-jenkins
     openshift.io/display-name: Fedora CoreOS Pipeline
     openshift.io/documentation-url: https://github.com/coreos/fedora-coreos-pipeline
     openshift.io/support-url: https://github.com/coreos/fedora-coreos-pipeline
-    openshift.io/provider-display-name: Fedora CoreOS
-    tags: fcos,jenkins,fedora
-  name: fedora-coreos
+    openshift.io/provider-display-name: fedora CoreOS
+    tags: ${TAGS}
+  name: ${DISTRO}-coreos
 parameters:
   - description: Git source URI for Jenkinsfile
     name: REPO_URL
     value: https://github.com/coreos/fedora-coreos-pipeline
   - description: Git branch/tag reference for Jenkinsfile
     name: REPO_REF
     value: master
+  - description: Distribution name
+    name: DISTRO
+    value: fedora
+  - description: Tags to label the pipeline with
+    name: TAGS
+    value: fedora,jenkins,fcos
+  - description: Name of the namespace to use
+    name: NAMESPACE
+    value: fedora-coreos
+  - description: Size of persistant storage
+    name: VOLUME_SIZE
+    value: 50Gi
 objects:
 
   ### JENKINS MASTER ###
 
   # use our own "gated" imagestream for the Jenkins master so we can test new
   # images before upgrading
-  - apiVersion: v1
+  - apiVersion: "image.openshift.io/v1"
```

Comment:
Hmm, why these `apiVersion` changes? Did you check that this still works against v3.6?

Reply:
Using a newer version of `oc`, it would fail without having the scoped version. I haven not tried it against 3.6. In my test, 3.11 would fail without scoping it.